docs/src/trace-viewer.md

```diff
@@ -222,3 +222,11 @@ playwright show-trace https://example.com/trace.zip
 ```bash csharp
 pwsh bin\Debug\netX\playwright.ps1 show-trace https://example.com/trace.zip
 ```
+
+## Viewing remote Traces on trace.playwright.dev
```

Comment:
- Let's maybe call this "Using trace.playwright.dev" and describe both 1) drag&drop local file and 2) pass a url?
- Please include a clickable link that goes to trace.playwright.dev here :)